site/_posts/2021-04-13-oggeh-fusionauth-gluu.md

```diff
@@ -0,0 +1,92 @@
+
+> [FusionAuth] works like a charm from the first attempt. That's something that doesn't usually happen when trying new things. 
+
+**Dan:** Tell me more about the Cloud CMS. Does it deliver JSON data which can then be used by a variety of clients (mobile, webapp, etc) in whatever way makes sense?
+
+**Ahmed:** Yes, that's exactly what it does. There's a [single-page](https://docs.oggeh.com) that covers our API reference, developer access, service architecture, and more. We have [a short video](https://dev.oggeh.com/) that presents OGGEH Cloud Platform for the developer community as well.
+
+**Dan:** How do you use FusionAuth? OAuth? User management? Social sign-on? Something else?
+
+**Ahmed:** As we speak, FusionAuth manages and authenticates our users to use OGGEH Cloud Platform tools (Cloud Apps, Cloud CMS, Cloud Stats and others). As well as a separate tenant/customer for each cloud app that requires its user authentication at the frontend (like e-commerce apps).
+
+We prefer OpenID Connect and OAuth 2.0 for the implementation. At the moment, we use both Authorization Code and Implicit flows for the authentication of our platform tools, and only Implicit flow on cloud apps (since they're progressive web apps).
+
+Securing OGGEH Cloud Platform customers are one of our main focuses, so we do not allow any special markup and/or scripts anyway. As I mentioned earlier, we introduce pure content management only. This means there's no easy way to inject code using our platform.
+
+**Dan:** Are you creating the tenants for each customer via the API or in the UX? Is the main reason for that separate because of the look and feel differences or are there other reasons to have a tenant for each cloud app?
+
+**Ahmed:** We prefer the API since it's part of automated builds. It's not because of the look and feel of FusionAuth (that's completely fine). It's more about abstracting complex configurations when creating our Cloud Apps. We aim to provide some sort of bare minimum UI for our customers to stick to pure content management, which includes user management. We're providing technical support anyway.
+
+**Dan:** What problems did we solve for you? And how were you solving them before FusionAuth?
+
+**Ahmed:** Two words: customization, and scalability! FusionAuth provides a very comforting way to organize tenants, applications, identify providers, email templates, and themes. In addition to the fact that the footprint is remarkably small and does not require great CPU or memory resources to operate. Which fits perfectly into our scalable stack.
+
+Earlier, we had to replicate the authentication server for each customer (Cloud App) just to customize the login theme and email templates. We also had to use separate servers on each due to the high resource requirements to start and operate. Scaling servers is far more complex and comes with a greater financial cost.
+
+> Literarily 40% of our expenses [are] just gone [after adopting FusionAuth]. 
```

Comment:
🎉  from the ⛰️ 's

Reply:
In fact, I think this should be a banner stat!

Reply:
Awesome point, I'll change the description. We do lead with this on the customers page.